bazel/upb_proto_library.bzl

```diff
@@ -99,7 +99,8 @@ def _cc_library_func(ctx, name, hdrs, srcs, dep_ccinfos):
             linking_context = linking_info.linking_context,
         )
 
-    if not versions.is_at_least("0.25.2", bazel_version):
+    # Assume Bazel prerelease versions (version == "") are ok.
+    if not versions.is_at_least("0.25.2", bazel_version) or bazel_version == "":
```

Comment:
Shouldn't it be `if not_fresh_enough and not_dev_build?`

Reply:
Ah yes, I coded that too quickly. :) Done.